Bottom: 84d4864d99f15408e57cb36e1a2ca9eabf15e549
Top:    c14a481100376feff50d71ccf263128da56e8d60
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-22 18:26:52 +0000

Refresh of add-netbyname-to-the-select-ac

---

diff --git a/src/select.c b/src/select.c
index 96d2fb9..bd4f373 100644
--- a/src/select.c
+++ b/src/select.c
@@ -1022,17 +1022,18 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
 
       MENU_LOOP (&PCB->NetlistLib);
       {
-        Cardinal i;
-        LibraryEntryType *entry;
-        ConnectionType conn;
+	Cardinal i;
+	LibraryEntryType *entry;
+	ConnectionType conn;
 
 	/* Name[0] and Name[1] are special purpose, not the actual name*/
-        if (menu->Name && REGEXEC (menu->Name + 2))
-          {
-            for (i = menu->EntryN, entry = menu->Entry; i; i--, entry++)
-              if (SeekPad (entry, &conn, false))
-                RatFindHook (conn.type, conn.ptr1, conn.ptr2, conn.ptr2, true, true);
-          }
+	if (menu->Name && menu->Name[0] != '\0' && menu->Name[1] != '\0' &&
+	    REGEXEC (menu->Name + 2))
+	  {
+	    for (i = menu->EntryN, entry = menu->Entry; i; i--, entry++)
+	      if (SeekPad (entry, &conn, false))
+		RatFindHook (conn.type, conn.ptr1, conn.ptr2, conn.ptr2, true, true);
+	  }
       }
       END_LOOP;
       RestoreUndoSerialNumber ();
